tests/dft-custom-hybrid/input.dat

```diff
@@ -0,0 +1,61 @@
+# DFT (hybrids) test of implementations in: hybrid_superfuncs.py
+
+##ORCA v.4.0.1, cc-pVDZ, RI-MP2, no FC, GRID7
+# PW6B95
+# WB97X-D3
+# HF-3c
+# PBEh-3c
+# PBE0
+##Gaussian 09: 5D SP Int(Grid=99770)
+# wb97x
+
+# MISSING: 
+#          "wpbe"     : build_wpbe_superfunctional,
+#          "wpbe0"    : build_wpbe0_superfunctional,
+#          "b5050lyp" : build_b5050lyp_superfunctional,
+
+psi4.core.set_output_file("output.dat", False)
```

Comment:
Are these output file resets leftovers or still doing something?

Reply:
I took it initially from the libxc test. Wasn't sure if i needed to use it. Just delete them?

Reply:
Yes, delete them. That libxc is a psiapi test masquerading as psithon. Yours are normal psithon.

Reply:
ah ok! Will do so.